spec/flattenizerSpec.js

```diff
@@ -0,0 +1,227 @@
+'use strict';
+import {flatten, unflatten} from '../src/flattenizer';
+
+describe('Flattenator!', () => {
+    describe('#flatten', () => {
```

Comment:
`#` implies method (in JSDoc). I would use `.` since it's a module (that is a namespace) containing functions.